internal/renderers/skia/opengl_surface.rs

```diff
@@ -28,8 +28,18 @@ impl super::Surface for OpenGLSurface {
         window: &dyn raw_window_handle::HasRawWindowHandle,
         display: &dyn raw_window_handle::HasRawDisplayHandle,
         size: PhysicalWindowSize,
-    ) -> Self {
-        let (current_glutin_context, glutin_surface) = Self::init_glutin(window, display, size);
+    ) -> Result<Self, PlatformError> {
+        let width: std::num::NonZeroU32 = size.width.try_into().map_err(|e| {
+            format!("Attempting to create window surface with width that doesn't fit into non-zero U32: {e}")
```

Comment:
Given that this is going to be shown to the user, i think it should not be so technical. Maybe something like
"Attempting to create window surface with an invalid size : {size:?}"
or
"Attempting to create window surface with an invalid with : {}", size.width()

Reply:
Ok, I'll pick the second one :)